build/jenkins_build.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 set -o errexit
 sh build.sh
+sh build_rpm.sh ${BUILD_NUMBER}
 mkdir -p /misc/babylon/Scratch/IBEX/Linux
+rm -f /misc/babylon/Scratch/IBEX/Linux/ibex-client-7.1-*.el7.x86_64.rpm
+cp -f ~/rpmbuild/RPMS/x86_64/ibex-client-7.1-${BUILD_NUMBER}.el7.x86_64.rpm /misc/babylon/Scratch/IBEX/Linux
+rm -f ~/rpmbuild/RPMS/x86_64/ibex-client-7.1-${BUILD_NUMBER}.el7.x86_64.rpm
```

Comment:
These have hard-coded release numbers. What should these read on the non-release builds?

Reply:
I hard coded the number as there is no linux release build to pick up a number. I have changed it to use version 0.0 and build_id-git_id as the package release details